api/src/org/apache/cloudstack/api/ApiConstants.java

```diff
@@ -639,6 +639,7 @@
     public static final String READ_ONLY = "readonly";
     public static final String SUPPORTS_REGION_LEVEL_VPC = "supportsregionLevelvpc";
     public static final String SUPPORTS_STRECHED_L2_SUBNET = "supportsstrechedl2subnet";
+    public static final String SUPPORTS_PUBLIC_ACCESS = "supportspublicaccess";
```

Comment:
Guest VM's with public network access is possible by checking default egress rule. Without adding this new notion of a 'public access', networks can reuse the `Egress Default Policy`.